packages/axgen/src/cmd/vector_store/teardown.ts

```diff
@@ -6,6 +6,7 @@ import { Pinecone } from '../../vector_stores/pinecone';
 import { Qdrant } from '../../vector_stores/qdrant';
 import { PgVector } from '../../vector_stores/pgvector';
 import { SUPPORTED_VECTOR_STORES, type SupportedVectorStores } from '../../vector_stores';
+import { Epsilla } from 'src/vector_stores/epsilla';
```

Comment:
This import should be a relative import
```suggestion
import { Epsilla } from '../../vector_stores/epsilla';
```